File: problems/two-sum/boilerplate-full/function.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>

##USER_CODE_HERE##

int main() {
    // Read input from standard input (stdin)
    std::string input;
    std::vector<int> inputs;

    while (std::getline(std::cin, input)) {
        std::istringstream iss(input);
        int value;
        while (iss >> value) {
            inputs.push_back(value);
        }
    }

    if (inputs.size() < 2) {
        std::cerr << "Insufficient input values." << std::endl;
        return 1; // Error code
    }

    int result = sum(inputs[0], inputs[1]);
    std::cout << result << std::endl;
    return 0;
}

File: problems/two-sum/boilerplate-full/function.java
import java.util.*;

public class Main {

    ##USER_CODE_HERE##

    public static void main(String[] args) {
        // Read input from standard input (stdin)
        Scanner scanner = new Scanner(System.in);
        List<Integer> inputs = new ArrayList<>();

        // Read integers from input
        while (scanner.hasNextInt()) {
            inputs.add(scanner.nextInt());
        }

        if (inputs.size() < 2) {
            System.err.println("Insufficient input values.");
            return; // Exit with an error
        }

        // Call the sum function with the first two inputs
        int num1 = inputs.get(0);
        int num2 = inputs.get(1);
        int result = sum(num1, num2);
        System.out.println(result);
    }
}
